data.sql

```diff
@@ -12,3 +12,48 @@ INSERT INTO animals VALUES (5, 'Charmander', '2020-02-08', 0, false, -11),
   (9, 'Boarmon', '2005-06-07', 7, true, 20.4),
   (10, 'Blossom', '1998-10-13', 3, true, 17),
   (11, 'Ditto', '2022-05-14', 4, true, 22) ;
+
+
+-- day 3
+-- Insert the following data into the owners table:
+-- Sam Smith 34 years old.
+-- Jennifer Orwell 19 years old.
+-- Bob 45 years old.
+-- Melody Pond 77 years old.
+-- Dean Winchester 14 years old.
+-- Jodie Whittaker 38 years old.
```

Comment:
-  [ ] I suggest you take this portion of the comments from your code. Details like this should be in your README and pull request. This makes sure your codebase is not clustered and looks very simple, clean yet professional 👍